src/components/Header.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { FaSearch } from 'react-icons/fa';
+import logo from '../utilities/logo.png';
+
+export default function Header(data) {
```

Comment:
As you are only using the `setMainPage` you can change it to 
```suggestion
export default function Header({ setMainPage }) {
```